### PR TITLE
[11] Create Input for a picture

### PR DESCRIPTION
**Pull request by catastrxphic**
Created a function that accepts images from the local library, as well as a function that let the user preview the image once selected, if the image is changed the preview changes with it. When the user goes back on the form the last image preview stays.

**Type of change**

- [ ] Bug Fixes
- [ ] Breaking Change
- [x] New Feature

**Have you done the following?**

- [x] Pull from staging/master
- [x] document your changes above
- [x] assigned a code reviewer
- [x] Deleted the comments in this template

### CODE REVIEW

**Comment by leonkoech** [comment]
Hello @catastrxphic , is this still WIP?

**Comment by catastrxphic** [comment]
hey @leonkoech, no. I just put it literally as the name on the project Github, sorry. :)

**Comment by leonkoech** [comment]
@catastrxphic haha that's okay. I changed the title to reflect the ticket number

File: pages/auth.tsx [hunk 1]
```diff
@@ -138,7 +160,17 @@ const Authentication = () => {
...
           <h1>add image</h1>
-          <input type="image" name="name" id="name"></input>
+          {/* display string base64 for url as an image and fit image with good resolution */}
+          <img src={preview} style={{objectFit:"contain"}} width='50%' height='50%' />
+              {/* get url of image when it is selected and/or changed */}
+              <input type={"file"} accept="image/*" onChange={(event)=>{ const file = event.target.files![0]
+              if (File){
+                setImage(file);
+              }else{
+                error;
```
Comment (leonkoech) [inline pages/auth.tsx:170]: the error function should be an await error(code). We need to add the error code for this particular instance in the error file with a code. @catastrxphic

**Review by leonkoech — APPROVED** [review]
Approving for now, but we need to address the error component

**Comment by leonkoech** [comment]
Also @catastrxphic Pul from staging and resolve any conflict before merging as it may break things considering staging was updated in the latest PR

**Review by leonkoech — CHANGES_REQUESTED** [review]
Pull from staging first

**Comment by leonkoech** [comment]
Also you did an amazing job! congrats you're promoted to assistant technical director XD

**Review by leonkoech — APPROVED** [review]

**Pull request merged**